_data/simple-icons.json

```diff
@@ -6954,7 +6954,7 @@
         },
         {
             "title": "Leanpub",
-            "hex": "FFFFFF",
+            "hex": "000000",
```

Comment:
No real use of black for their website. If we want to use a darker color instead of white, we might use the color of their header and footer (`#222222`).